docs/en/users/03_Main_view.md

```diff
@@ -1,14 +1,36 @@
+FreshRSS support has three primary viewing modes: Normal, Global, and Reader view.
+
 # Normal view
 
-**TODO**
+Normal view will allow you to view articles in a compressed view. They can be separated by category or individual feed, or viewed in the "main stream" containing all feeds. Clicking a feed in the sidebar (mobile users will need to click the folder icon to open it) will open that feed's view. 
+
+## Article List
+
+By default, the normal view includes six items per article. From left to right:
+* **Read status:** An envalope icon to show if the article has been read or not. Closed envalopes are unread, open envalopes are read. Clicking on the icon will toggle the read status.
+* **Favourite status:** A star icon to show if the article has been favourited or not. Filled stars are favourited, empty stars are not. Clicking on the icon will toggle the favourite status.
+* **Feed name:** The name of the feed that the article is from. Clicking the feed name will move to that feed's view in normal view.
+* **Article title:** The title of the article. Clicking will open the article for viewing within FreshRSS. 
+* **Article date/time:** The time the article was posted.
+* **Link to original article:** A globe icon that can be clicked to go to the article on the original website.
+
+## Normal View Sidebar
+
+Clicking the gear icon next to an individual feed will display additional options for that feed. 
+* **Filter:** Run the defined filter to mark articles as read
+* **Statistics:** View statistics about the feed
+* **See website:** Open the feed's website in another tab
+* **Manage:** Configure the feed
+* **Actualize:** Force-update the feed
+* **Mark as read:** Mark all items in the feed as read
 
 # Global view
 
-**TODO**
+Global view allows quick views of many feeds' statuses at once. Feeds and categories are shown with the number of unread articles next to them. Clicking a stream will open it in a view similar to normal view. 
```

Comment:
`feed's statuses` instead of `many feeds' statuses` and replace `Clicking a stream` by `Clicking on a feed's name`!